mbed_tools_ci_scripts/create_news_file.py

```diff
@@ -0,0 +1,72 @@
+"""Easy news files generation.
+
+Usage:
+    create-news-file "Fixed a bug" --type bugfix
+"""
+import argparse
+import enum
+import pathlib
+from datetime import datetime
+
+from mbed_tools_ci_scripts.utils.configuration import configuration, ConfigurationVariable
+
+
+NEWS_DIR = configuration.get_value(ConfigurationVariable.NEWS_DIR)
+
+
+class NewsType(enum.Enum):
+    """Describes the type of news we're writing."""
+
+    BUGFIX = 0
```

Comment:
We could just set the `enum` values to the actual extension string we want, then we can look it up with `Enum.FIELD.value`. I think that might be cleaner than having this extra `extension` property?

Reply:
That makes perfect sense.

Reply:
Fixed https://github.com/ARMmbed/mbed-tools-ci-scripts/pull/34/commits/ed604ddb0b8ca3466ed634812a501c4e451b1b1b.